tests/tensorflow/quantization/test_algorithm_quantization.py

```diff
@@ -190,8 +192,8 @@ def test_export_saturatuion_fix(disabled):
 
     ref_weight_qspec = TFQuantizerSpec(mode=QuantizationMode.SYMMETRIC,
                                        num_bits=8,
-                                       signedness_to_force=None,
-                                       per_channel=False,
+                                       signedness_to_force=True,
+                                       per_channel=True,
```

Comment:
Could you provide a short explanation for the changes?

Reply:
As above.